module/motion/OldWalkEngine/src/OldWalkEngine.h

```diff
@@ -165,20 +163,20 @@ namespace motion {
         float step_height_fast_fraction;
         double phase1Single;
         double phase2Single;
-        arma::vec2 footOffset;
-        double legYaw;
```

Comment:
Why did this change?

Reply:
The only place it is used is to create a std::vector of <ServoID, float> so everything else around that part is float, including the function in which it is used. Rather than casting it for its only use I changed its definition.